mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/rendering.kt

```diff
@@ -81,6 +81,8 @@ internal class AnsiRendering(
 		return stringBuilder.apply {
 			clear()
 
+			append(ansiBeginSynchronizedUpdate)
```

Comment:
So glad these are here now. It pained me to do all this `StringBuilder` stuff and then have a second `StringBuilder` just to wrap these around it.

At some point I want to experiment with switching the whole pipeline to deal in bytes rather than chars/Strings.